DuckDuckGo/PasswordManager/Bitwarden/Model/BWManager.swift

```diff
@@ -215,7 +216,8 @@ final class BWManager: BWManagement, ObservableObject {
                 status = .notRunning
             }
         default:
-            logOrAssertionFailure("BWManager: Wrong handler")
+            Logger.bitWarden.fault("BWManager: Wrong handler")
+            assertionFailure("BWManager: Wrong handler")
```

Comment:
perhaps we could write our own faultAndAssert function for Logger, so we don't have to write fault and assertion calls twice, here and multiple times below?

Reply:
Logs should always be done "in place" and not wrapped in functions, wrapping the log in a function removes it from the context and invalidates some cool logging features used in crash reports and system diagnostics